src/main/java/com/odysseusinc/arachne/portal/model/ParticipantRole.java

```diff
@@ -70,7 +71,8 @@
             LIMITED_EDIT_PAPER,
             ACCESS_PAPER,
             EDIT_ANALYSIS,
-            EDIT_INSIGHT
+            EDIT_INSIGHT,
+            UPDATE_SUBMISSION,
```

Comment:
We discussed that submissions hiding will be available not only for Leads, but Analysis owners